style.json

```diff
@@ -439,6 +442,44 @@
         "fill-opacity": 0.8
       }
     },
+    {
+      "id": "landuse-aerodrome",
+      "type": "fill",
+      "metadata": {
+        "mapbox:group": "1444849388993.3071",
+        "taxonomy:group": "landuse"
+      },
+      "source": "basemap",
+      "source-layer": "aeroway",
+      "filter": [
+        "in",
+        "class",
+        "aerodrome"
+      ],
+      "paint": {
+        "fill-color": "#8665E7",
+        "fill-opacity": 0.06
+      }
+    },
+    {
+      "id": "landuse-bus",
+      "type": "fill",
+      "metadata": {
+        "mapbox:group": "1444849388993.3071",
+        "taxonomy:group": "landuse"
+      },
+      "source": "basemap",
+      "source-layer": "landuse",
+      "filter": [
+        "in",
+        "class",
+        "bus_station"
+      ],
+      "paint": {
+        "fill-color": "#8665E7",
+        "fill-opacity": 0.06
+      }
+    },
```

Comment:
there are still a few POI polygons missing: theme_park and zoo (low opacity pink ?), stadium pitch and playground (low opacity Unclassified colour ?)